docs/ref/django-admin.txt

```diff
@@ -222,6 +222,13 @@ program manually.
 
 Specifies the database onto which to open a shell. Defaults to ``default``.
 
+.. note::
+
+    Some of `DATABASES['OPTIONS']` are not passed to the dbshell environment.
+    For example even though isolation level is set to``REPEATABLE READ`` or
+    ``SERIALIZABLE`` in `DATABASES['OPTIONS'], but in the client prompt only
+    the default option i.e ``READ COMMITTED`` will be available.
```

Comment:
This comment about isolation level is not precise (see [comment](https://code.djangoproject.com/ticket/30865#comment:9)). I don't think that we need it. Also we should add a reference to the settings documentation. Maybe:
```
.. note::

    Be aware that not all options set it in the :setting:`OPTIONS` part of your
    database configuration in :setting:`DATABASES` are passed to the
    command-line client, e.g. ``'isolation_level'``.
```

Reply:
@felixxm I have changed the wordings a bit made it a little dramatic 😛 hope you don't mind.
Also if everything is okay I can squash commits.

Reply:
I removed an exclamation mark. We don't need to shout on users.